docs/src/explanations/annotations.md

```diff
@@ -134,10 +134,8 @@ class ModC(widthC: Int) extends Module {
 
 Compiling this circuit to Verilog will then result in the `InfoTransform` running and the added `println`s showing information about the components annotated.
 
-```scala mdoc
+```scala mdoc:compile-only
 import chisel3.stage.{ChiselStage, ChiselGeneratorAnnotation}
 
-// This currently doesn't work because of mdoc limitations. However, it will work
-// in your normal Scala code.
-//(new ChiselStage).execute(Array.empty, Seq(ChiselGeneratorAnnotation(() => new ModC(4))))
+(new ChiselStage).execute(Array.empty, Seq(ChiselGeneratorAnnotation(() => new ModC(4))))
```

Comment:
ok, this is the thing I was thinking of. The original code was commented out, so I wanted to switch it to just be compile-only so at least it is compile checked. I tried to get the example on this page to run in scastie: 
https://scastie.scala-lang.org/yN7nUBoWQ5CKvcV4UoGkwg
but I still get an exception. Is there something missing in the example to make it work?

Reply:
Note: as committed, this stuff DOES properly generate the docs, but I'm worried the example is flawed if it doesn't run in scastie.

Reply:
I don't think this is an issue with `mdoc`, I think it's actually a manifestation of https://github.com/chipsalliance/chisel3/issues/1803 (which is closed on master and 1.4.x but not yet on a published version).

Reply:
ok cool, then I will not block the PR on seeing this run.